### DISCUSSION TITLE
q-img slow loading

### DISCUSSION THREAD

**Discussion by viniciusccarvalho** [quasarframework/quasar#14968] [General - Components / Directives / etc]
Hi folks, something very odd started on my project. I know I am to blame, but I can't find the root cause.

I have two images that are loaded dynamically as the user changes via a pagination control. They have always loaded blazing fast on my local machine. The server response time is less than 15ms for each request (I checked both chrome network and via server logs) so the images are fetched quite fast.

However, for some reason I really can't pinpoint, after today and some changes (none touched the images dom objects) the images now take over 1 minute to load at a times. When the page changes, and the image which is computed is called, there's  loading spin on the image dom, and that stays for several seconds (up to 135s once). Looking at networking and server the image has been served to the browser, so what could be causing this slowness?

Some segments of the code here:
```html
 <q-img :src="scans[currentPage]" class="scan-image" fit="contain" 
                                    style="max-height: 95vh">
                                </q-img>

```
```javascript
axios.get("/api/magazines/" + this.selectedMagazine.id).then(response => {
                        this.selectedMagazine = response.data;
                        this.currentPage = 0;
                        this.scans = []
                        this.selectedPage = this.selectedMagazine.pages[0]
                       
//scans arrays contains a list with the static resources of each page scan
 this.selectedMagazine.pages.forEach(page => {
                            let file = page.scans.filter(scan => scan.type == "FULL").map(uri => "/static/images/" + this.selectedPage.publicationId + "/" + uri.file)[0]
                            this.scans.push(file)
                        })

                    })
                    
             
```

I have tried to add `loading="lazy"` without success.

After the problem has manifested I switched from the prod js to dev one, and noticed a few warning on vuejs (not sure if it is related, but adding here as contextual data)

```
[Vue warn]: Unhandled error during execution of render function 
  at <QPageContainer> 
  at <QLayout view="hHh lpR fFf" > 
  at <App>```

Any ideas on what have I done to cause this? Where could I look for causes and hints?

Thank you

**Comment by pdanpdan** [comment]
I would expect there is an error inside QPageContainer - expand the error log in the console and see where it is happening

**Reply by viniciusccarvalho** [reply]
Thank you @pdanpdan problem was with chrome after all. Glad it was an easy way out

**Comment by viniciusccarvalho** [comment]
Ok folks, never mind it was a chrome issue. I applied the update that it was requesting and the problem went away. The fact that the network console showed it loading so quick made me think chrome was not a problem, should have known better.
